in_text/text_04_predictions.yml

```
@@ -2,7 +2,7 @@ title: >-
   Data to support network-wide 7-day ahead forecasting of water temperature in the Delaware River Basin: 4) model predictions
 
 abstract: >-
-  This data release contains the forcings and outputs of 7-day ahead maximum water temperature forecasting models
+  This data release contains the forcings and outputs of 7-day ahead maximum water temperature forecasting models 
```

are any of the data in here actually considered forcings?
```suggestion
  This data release contains the forcings and outputs of 7-day ahead maximum water temperature forecasting models
```